node/clients/rewriter.ts

```diff
@@ -1,84 +1,58 @@
 import { AppGraphQLClient, InstanceOptions, IOContext } from '@vtex/api'
-import { path } from 'ramda'
 
-export interface RouteIndexFiles {
-  lastChangeDate: string
-  routeIndexFiles: RouteIndexFileEntry[]
-}
-
-export interface RouteIndexFileEntry {
-  fileName: string
-  fileSize: number
-}
-
-export interface RouteIndexEntry {
+export interface Internal {
+  from: string
+  type: string
   id: string
-  lastChangeDate: string
+  endDate?: string
+  imagePath?: string
+  imageTitle?: string
 }
 
-export interface Internal {
-  from: string
-  declarer: string // vtex.store@2.x
-  type: string // product
-  id: string // 123
-  query: JSON | null // {subtype: 'default'}
-  bindings: string[] | null // binding where this route is appliable
-  endDate: string | null // When the internal route expires
-  imagePath: string | null
-  imageTitle: string | null
+export interface ListInternalsResponse {
+  routes?: Internal[]
+  next?: string
 }
 
 export class Rewriter extends AppGraphQLClient {
-  public constructor(ctx: IOContext, opts?: InstanceOptions) {
+  constructor(ctx: IOContext, opts?: InstanceOptions) {
     super('vtex.rewriter', ctx, opts)
   }
 
-  public routesIndexFiles = (): Promise<RouteIndexFiles> =>
+  public listInternals = (
+    limit: number,
+    next: Maybe<string>
+  ): Promise<ListInternalsResponse> =>
     this.graphql
-      .query<string[], {}>(
+      .query<
+        { internal: { listInternals: { next: string; routes: Internal[] } } },
+        { next: Maybe<string>; limit: number }
+      >(
         {
           query: `
-      query RoutesIndexFiles {
+      query ListInternals($limit: Int, $next: String) {
         internal {
-          indexFiles {
-            routeIndexFiles {
-              fileName
-              fileSize
+          listInternals(limit: $limit, next: $next) {
+            routes {
+              from
+              type
+              endDate
+              imagePath
+              imageTitle
+              id
```

Comment:
It would be nice to also have a lastModified attribute as well. The format in the sitemap is this one:
`<lastmod>2020-03-24</lastmod>`

Reply:
There isn't a last modified attribute for the route in Rewriter, do you think we should add one?